client/src/components/user/dashboard.js

```diff
@@ -9,12 +10,27 @@ import DashboardCalendar from '../date/dashboardCalendar';
 import UpdateSubscription from '../subscription/updateSubscription';
 import CreateSubscription from '../subscription/createSubscription';
 import SubscriptionDetail from '../subscription/subscriptionDetail';
+import DashboardModal from '../dashboardModal';
 import logo from '../../assets/watering-can.png';
 
 const href = process && process.env && process.env.NODE_ENV === 'production'
   ? '/auth/logout'
   : 'http://localhost:5000/auth/logout';
 
+const waitUserClick = () => (
+  new Promise((resolve, ) => {
```

Comment:
this looks like it only needs one argument

```suggestion
  new Promise((resolve) => {
```